arkane/modelchem.py

```diff
@@ -33,12 +33,31 @@
 
 from __future__ import annotations
 
+import logging
 from dataclasses import dataclass, Field, fields, MISSING, replace
 from typing import Iterable, Union
 
 from rmgpy.rmgobject import RMGObject
 
 
+def standardize_name(name: str) -> str:
+    """Remove spaces, hyphens, and make lowercase"""
+    if isinstance(name, str):
+        return name.replace('-', '').replace(' ', '').lower()
+    else:
+        return name
+
+
+METHODS_THAT_REQUIRE_SOFTWARE = [
+    'B97-D',
+    'B97-D3',
+    'wB97X-D',
+    'wB97X-D3',
+    'wB97M-V',
+]
```

Comment:
Should this be included in the database instead? Either in data.py of quantum corrections or a new file